src/app/zap-templates/templates/app/im-cluster-command-handler.zapt

```diff
@@ -99,74 +99,79 @@ void Dispatch{{asUpperCamelCase side}}Command({{#if (isServer side)}}CommandHand
 
 } // namespace Clusters
 
-void DispatchSingleClusterCommand(ClusterId aClusterId, CommandId aCommandId, EndpointId aEndPointId,
-                             TLV::TLVReader & aReader, CommandHandler * apCommandObj)
+void SetInvalidClusterId(const ConcreteCommandPath & aCommandPath, Command * apCommandObj)
```

Comment:
A followup: move this to somewhere that's not generated, since it's just code that does not depend on any of the generation bits.

Reply:
I finally removed this method since with #10125 there was not much duplicated code anymore.